files/es/web/javascript/reference/operators/operator_precedence/index.md

```diff
@@ -22,8 +22,8 @@ con la precedencia más alta va primero y la asociatividad no importa. Observe c
 tiene mayor precedencia que la suma y se ejecuta primero, a pesar de que la suma se escribe primero en el código.
 
 ```js
-console.log(3 + 10 * 2);   // muestra 23
-console.log(3 + (10 * 2)); // muestra 23 porque los paréntesis son superfluos
+console.log(3 + 10 * 2); // muestra 23
+console.log(3 + 10 * 2); // muestra 23 porque los paréntesis son superfluos
 console.log((3 + 10) * 2); // muestra 26 porque los paréntesis cambian el orden
 ```
```

Comment:
Just a note, thats for the example

````
```js-nolint
console.log(3 + 10 * 2); // muestra 23
console.log(3 + (10 * 2)); // muestra 23 porque los paréntesis son superfluos
console.log((3 + 10) * 2); // muestra 26 porque los paréntesis cambian el orden
```
````